### PR TITLE
Feature list club favorites

### LINKED ISSUE #21: Regular user can see a list of favorite clubs
Description: 

**As a** regular user,
**I can** see a list of favorite clubs 
**so that** I can enter there without losing time.

Empty state: 
**If** there is no clubs in my “favorite clubs” list, 
**then** show “You have no added any favorite club, yet”.

<div display="flex" align="center">
Home
Favorites
</div>

### PR DESCRIPTION
**Pull request by condef5**
### Status
Required

### Related Issues
closes #21 

### Implementation Notes
* User can see the list favorite clubs.

### CODE REVIEW

File: client/src/services/club.js [hunk 1]
```diff
@@ -26,6 +26,15 @@ async function getClubs() {
   return response.json();
 }
 
+async function getFavoriteClubs() {
+  const response = await fetch(`${apiUrl}/clubs`, {
```
Comment (cristianbgp) [inline client/src/services/club.js]: I think this endpoint should be filtered

Comment (condef5) [reply client/src/services/club.js]: Yep, you right, I will fix right now.

File: client/src/components/navbar.js [hunk 2]
```diff
@@ -1,41 +1,80 @@
...
 import { Link } from "@reach/router";
 
 function Navbar() {
+  const styleMenu = {
+    textDecoration: "none",
+    color: "inherit",
+    padding: "15px",
+    borderRight: "1px solid #ddd",
+    flex: "auto",
+    ":last-child": {
```
Comment (liannivin) [inline client/src/components/navbar.js]: What it does?

Comment (liannivin) [inline client/src/components/navbar.js]: Could you explain how `:last-child` works, please?

**Pull request merged**